packages/salesforcedx-vscode-core/src/commands/forceStopApexDebugLogging.ts

```diff
@@ -65,16 +65,12 @@ export async function restoreDebugLevels(): Promise<void> {
   }
 }
 
-function getRestoreLevelsCommand(): Command {
+function deleteTraceFlag(): Command {
   return new SfdxCommandBuilder()
     .withDescription(nls.localize('force_stop_apex_debug_logging'))
-    .withArg('force:data:record:update')
-    .withFlag('--sobjecttype', 'DebugLevel')
-    .withFlag('--sobjectid', developerLogTraceFlag.getDebugLevelId())
-    .withFlag(
-      '--values',
-      `ApexCode=${developerLogTraceFlag.getPrevApexCodeDebugLevel()} Visualforce=${developerLogTraceFlag.getPrevVFCodeDebugLevel()}`
-    )
+    .withArg('force:data:record:delete')
+    .withFlag('--sobjecttype', 'TraceFlag')
+    .withFlag('--sobjectid', developerLogTraceFlag.getTraceFlagId()!)
```

Comment:
Changing command to delete the traceflag instead of updating the levels

Reply:
Using the `!` since because we know it's set because to reach this it passes the parameter gatherer.